mscore/musescore.cpp

```diff
@@ -6314,6 +6314,14 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   _styleDlg->setScore(cs);
             _styleDlg->exec();
             }
+      else if (cmd == "edit-header-footer") {
+            if (!_styleDlg)
+                  _styleDlg = new EditStyle{ cs,this };
```

Comment:
Space between `EditStyle` and `{`please

Reply:
Alright

Reply:
And then squash the commit, we'd want one commit only for this PR

Reply:
Okay